.github/ci_templates/build.yml

```diff
@@ -8,9 +8,9 @@ steps:
 
   - name: Run go generate
     run: |
-      BASE_GOLANG_ALPINE={!{ printf "%s%s" (datasource "image_versions").REGISTRY_PATH (datasource "image_versions").BASE_GOLANG_ALPINE | quote }!}
-      docker run -v $(pwd):/deckhouse -v ~/go-pkg-cache:/go/pkg -w /deckhouse/tools ${BASE_GOLANG_ALPINE} go generate .
-      docker run -v $(pwd):/deckhouse -v ~/go-pkg-cache:/go/pkg -w /deckhouse/modules/500-upmeter/hooks/smokemini/internal/snapshot ${BASE_GOLANG_ALPINE} go generate .
+      BASE_GOLANG_18_ALPINE={!{ printf "%s%s" (datasource "image_versions").REGISTRY_PATH (datasource "image_versions").BASE_GOLANG_18_ALPINE | quote }!}
```

Comment:
I see a problem. We test 'go generate' using Golang 1.18, but build with Golang 1.16.

Reply:
Make sense update also building scripts. Thank you